scriptmodules/emulators/amiberry.sh

```diff
@@ -17,22 +17,24 @@ rp_module_section="opt"
 rp_module_flags="!x86 !mali !kms"
 
 function depends_amiberry() {
-    depends_uae4arm
+    getDepends libxml2-dev libflac-dev libmpg123-dev libpng12-dev libmpeg2-4-dev
 }
 
 function sources_amiberry() {
-    gitPullOrClone "$md_build" https://github.com/midwan/amiberry/
+    gitPullOrClone "$md_build" https://github.com/midwan/amiberry/ "dev"
```

Comment:
Just to check, but this is for testing now right (We will continue to track master).

Reply:
Yep. As I said, I'm also guessing the "-dev" suffix on the binary may also be gone when @midwan makes the switch to master. For now, the PR is just to help "kick the tires" and sort out potential issues with the new version.

Reply:
Yes, the "-dev" suffix will be gone once this is moved to master ;-)